src/runtime-rs/crates/resource/src/rootfs/block_rootfs.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::{Rootfs, ROOTFS};
+use crate::share_fs::{do_get_guest_path, do_get_host_path};
+use agent::Storage;
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+use hypervisor::{device::DeviceManager, BlockConfig, DeviceConfig};
+use kata_types::mount::Mount;
+use nix::sys::stat::{self, SFlag};
+use std::fs;
+use tokio::sync::RwLock;
+
+pub(crate) struct BlockRootfs {
+    guest_path: String,
+    device_id: String,
+    mount: oci::Mount,
+    storage: Option<agent::Storage>,
+}
+
+impl BlockRootfs {
+    pub async fn new(
+        d: &RwLock<DeviceManager>,
+        sid: &str,
+        cid: &str,
+        dev_id: u64,
+        _bundle_path: &str,
```

Comment:
`_bundle_path` is not used now, delete it?

Reply:
fixed